src/strategies/path-balance-staked-and-locked/schema.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$ref": "#/definitions/Strategy",
+  "definitions": {
+    "Strategy": {
+      "title": "Strategy",
+      "type": "object",
+      "properties": {
+        "symbol": {
+          "type": "string",
+          "title": "Symbol",
+          "examples": ["e.g. UNI"]
+        },
+        "address": {
+          "type": "string",
+          "title": "Contract address",
+          "examples": ["e.g. 0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984"]
+        },
+        "decimals": {
+          "type": "number",
+          "title": "Decimals",
+          "examples": ["e.g. 18"]
```

Comment:
There are other params missing in this? 🤔 Below are the params I see in examples.json
```
        "tokenAddress": "0x2a2550e0a75acec6d811ae3930732f7f3ad67588",
        "stakingAddress": "0x0bF6eeA205F46d796458D090F3aA333149287854",
        "lockedAddresses": [
          "0xDe48716A14C4CBc09656E21CE7F40FC1a02b3a25",
          "0x67f0260254FB3Cee97dA18077927888Ed72D1f17",
          "0xeE729DB66431e4401D63A38a2048d8CE0DF96eC3",
          "0x8923f2A0465287E2F8564F85a7f62ed34d947594",
          "0xfB05dc346f36eE88680e885CaC881983FAdEcBD4",
          "0x9A319F461A84EA5b508AE1e97113fD1743ED691D"
        ],
        "methodABI"
        ```

Reply:
have added the missing params